scripts/bnsd/genesis_app_state.json

```diff
@@ -17,6 +17,32 @@
       "coins": ["4000 BASH"]
     }
   ],
+  "account": {
+    "domains": [
+      {
+        "admin": "seq:test/alice/1",
+        "domain": "iov",
+        "valid_until": 1895827578
+      }
```

Comment:
Got it, you forgot to add has_superuser flag to the domain configuration.

Reply:
Ok and should it be true or false? The genesis file from iov-core does not have that flag.

Reply:
Checking right now. Then genesis for IOV Core should be fixed too.

Reply:
I've checked out some Slack convos and I think I should set it to true.

Reply:
If I set it to false, the same problem persists. When set to true, it says only admin can register.

Reply:
@davepuchyr looks like a bug in bnsd.

Reply:
The *iov domain is special because it's the only domain with `has_superuser = false`.  Without a superuser, certain operations are restricted, like the domain admin **cannot** delete accounts within the domain, only account owners can delete their account.  

The economic design of the starnames is that casual users can inexpensively use the *iov domain whereas hardcore users/businesses create domains with `has_superuser == true`.

Long story short, it's not a bug in bnsd.

Reply:
*iov being the only domain with `has_superuser = false` was my impression and your functionality descrition makes sense, but we are talking about **registering** an iovname. I believe that operation should be able to be done by any user.

Reply:
We may have to check iov-core then, since it's an error that happens when decoding.

Reply:
Yes, you are right.  We need to fix https://github.com/iov-one/iov-core/blob/master/packages/iov-bns/src/decodeobjects.ts.  https://github.com/iov-one/iov-core/pull/1415 is a PR that I'm anxious to merge and can include the fix.